### DISCUSSION TITLE
General requests

### DISCUSSION THREAD

**Discussion by WilsonBradley** [curif/AgeOfJoy-2022.1#24] [Ideas]
Would it be possible to compile and post the betas builds as you test them?

Example- most sites post them under Actions for rolling builds
![image](https://user-images.githubusercontent.com/8173893/205811210-0aa3b39d-7412-4d64-b9d8-cf290697c7df.png)

**Comment by curif** [comment]
Yes, but there aren't Actions for Unity I gess. So I will build and upload to GitHub new versions to test, and upload to itchio later.

In fact right now Meta breaks the development process of all the developers. I hope they can solve quickly: https://forums.oculusvr.com/t5/Get-Help/Known-Issue-Error-saying-quot-We-re-having-problems-loading-this/m-p/1004226#M178769
